src/client/components/definition/definitionData.js

```diff
@@ -0,0 +1,36 @@
+import {
+  DEFINITION_OPEN,
+  DEFINITION_HIGH,
+  DEFINITION_LOW,
+  DEFINITION_MARKET_CAP,
+  DEFINITION_VOLUME,
+  DEFINITION_DIVIDEND,
+  DEFINITION_PE_RATIO,
+  DEFINITION_ROE,
+  DEFINITION_FCFY
+} from '../../../constants';
+
+module.exports = Object.freeze({
+  [DEFINITION_OPEN]: 'Price of the security at the beginning of the trading day.',
```

Comment:
alphabetize

Reply:
you could almost do an array of glossary terms, e.g.
```
[
  {
    title: 'Dividend',
    definition: 'An amount of the company\'s profits...'
  },
  {
    title: 'FCFY',
    definition: 'A comparison between free cash flow per share and share price...'
  },
  ...
]
```
In this approach it would be easy to iterate over all of the definitions in alphabetical order in the component, since they're already in a set order in an array.  I don't know that we'd need the DEFINITION_ constants anymore in this approach; we'd just have `title` and `definition` fields on each object in the array.